README.md

```diff
@@ -72,7 +72,7 @@ maintained stacks.
 We value all positive contributions to the Docker stacks project,
 from [bug reports](https://jupyter-docker-stacks.readthedocs.io/en/latest/contributing/issues.html)
 to [pull requests](https://jupyter-docker-stacks.readthedocs.io/en/latest/contributing/packages.html)
-to help answering questions.
+to help to answer questions.
```

Comment:
"from bug reports to pull requests to help withto answer questions."

Maybe...

```suggestion
to help with answering questions.
```